.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+/test.md
```

Comment:
- [ ] [OPTIONAL] Since the test.md is a file and not a directory you don't need to add "/" here just the name `test.md` should work fine. 😉